CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# CONTRIBUTING GUIDELINES
+
+[日本語のコントリビューティングガイドライン](CONTRIBUTING_ja.md)
+
+## About of decomoji add or remove
+
+It does not accept in principle.
+
+Use decomoji-template.psd if you want to use new one.
+
+Remove files from `decomoji/` directory if you don't want to use one.
+
+import.rb auto-register those of `decomoji/` directory.
```

Comment:
`import.rb` rule them all to remove a registered decomoji on your Slack automatically.